rulebook-core/src/main/java/com/deliveredtechnologies/rulebook/model/runner/RuleAdapter.java

```diff
@@ -223,6 +226,45 @@ public void setResult(Result result) {
     return _rule.getResult();
   }
 
+  private List<Class<?>> eligibleParameterTypes(Field field) {
+    Type genericType = field.getGenericType();
+    if (genericType instanceof ParameterizedType) {
+      ParameterizedType parameterizedType = (ParameterizedType)genericType;
+      Type[] typeArguments = parameterizedType.getActualTypeArguments();
+      if (typeArguments != null && typeArguments.length > 0) {
```

Comment:
Should this be typeArguments.length == 1? Does it matter?

Reply:
Probably not. And since _== 1_ would have the same result, in situations like this, I generally prefer to specify a boundary instead of a hard equivalence.